### PR TITLE
feat(EA) restore Meetup OAuth flow

### PR DESCRIPTION
**Pull request by lucatume**
Ticket:
* https://central.tri.be/issues/125635
* https://central.tri.be/issues/130274

This PR restores the code, on EA Client, to interact with Meetup API using the OAuth authentication flow.

### CODE REVIEW

**Comment by lucatume** [comment]
I'm skipping some `views_integration` tests here I've fixed in https://github.com/moderntribe/the-events-calendar/pull/2615

**Comment by borkweb** [comment]
I'd suggest QA-ing in a branch

File: src/Tribe/Aggregator/Settings.php [hunk 1]
```diff
@@ -175,20 +176,175 @@ public function is_ea_authorized_for_eb() {
 		return true;
 	}
 
+	/*
+
+		=====================================================================================================
```
[the remark this answers is not in the document]
Comment (lucatume) [reply src/Tribe/Aggregator/Settings.php]: It's about the artist feeling of amazement.

**Comment by lucatume** [comment]
Closing this in favor of https://github.com/moderntribe/the-events-calendar/pull/2635 to rebase on `4.9.5.1`.

**Pull request closed without merging**